src/main/resources/fabric.mod.json

```diff
@@ -28,7 +28,7 @@
   "depends": {
     "fabricloader": "*",
     "fabric": "*",
-    "minecraft": ">=1.16"
+    "minecraft": ">=1.18.2"
```

Comment:
That's a bit too restrictive, I bet it still works on 1.16 with a newer cloth version.

Reply:
It might be too restrictive but 1.16 does not seem to be the right choice because I tried it with the latest Fabric API and Mod Menu builds for 1.16 and the Mod Menu integration crashes.

Reply:
Alright, I'll check this myself later.

Reply:
I was going to check the appropriate version later tonight, when I have time to get back to this PR and do what you asked.